eMPRess_GUI_version1.py

```diff
@@ -0,0 +1,354 @@
+# eMPRess_GUI_iteration5
+
+import tkinter as tk
+from tkinter import messagebox
+from pathlib import Path
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
+from matplotlib.backend_bases import MouseEvent, key_press_handler
+import empress
+
+class App:
+
+    def __init__(self, master):
+        self.master = master
+        ### configures the master frame ###
+        master.grid_rowconfigure(0, weight=1)
+        master.grid_rowconfigure(1, weight=2)
+        master.grid_columnconfigure(0, weight=1)
+        master.grid_columnconfigure(1, weight=1)
+
+        ### creates a logo frame on top of the master frame ###
+        self.logo_frame = tk.Frame(master)
+        self.logo_frame.grid(row=0, column=0, columnspan=2, sticky="nsew")
+        self.logo_frame.grid_propagate(False)
+
+        # adds background image
+        photo = tk.PhotoImage(file="JaneTransLogo-thin.png")
+        label = tk.Label(self.logo_frame, image=photo)
+        label.place(x=0, y=0)
+        label.image = photo
+
+        ### creates an input frame on the left side of the master frame ###
+        self.func_frame = tk.Frame(master)
```

Comment:
Why is this variable called `func_name`?

Reply:
It is named func_frame, i.e. function frame because it is the frame for those buttons (functionalities) on the left. There are also logo frame and the output frame. Or I can change this to input frame (input_frame). I think it will be better. :)